src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,7 +3,7 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
```

Comment:
let's revert this change and make it private again because we should follow the incapsulation principle from OOP(which we will cover later on in the course) and not give everybody access to our users, only via its public method `findByEmail()` which we should use in `AuthenticationService`